### DISCUSSION TITLE
Newly added Carla Vehicle Wheel Not Rotating

### DISCUSSION THREAD

**Discussion by HarishKumar051993** [carla-simulator/carla#4226] [General]
Hi Everyone, I have created a new 4 wheeled vehicle by referring to the documentation below:

https://carla.readthedocs.io/en/latest/tuto_A_add_vehicle/

After creating vehicle, i can able to move the vehicle but the wheels are not rotating instead the complete car is moving. Even after doing left/right of steering also the wheels are not moving.

Can anyone help on this why it is causing the problem?

**Comment by gsmario** [comment]
Hello @HarishKumar051993 
Please, check your animation blueprint, maybe some nodes are not connected or you didn't created it. 
Follow this 2 steps to configure the blueprint.
Thanks for taking the time to contact us.
![image](https://user-images.githubusercontent.com/44405709/118966312-e59e6600-b969-11eb-91d8-59bed169ea2f.png)

**Reply by HarishKumar051993** [reply]
@gsmario Thanks for the response. I have resolved the issue. The problem was at while creating the fbx in Maya. The wheels are not binded properly and instead the whole vehicle is binded. Now i binded separately the wheels and it is working now.

**Comment by anandkvmar** [comment]
Hey @gsmario @HarishKumar051993 ,
I’m working on importing my own vehicle model into Carla, but I’m facing an issue while running the vehicle in the Carla simulator. The vehicle doesn’t seem to be moving and the wheels are movies in a different axis. I have attached the relevant images and video of the issue. Can someone please look into the issue and help me find a fix.

https://youtu.be/3E--y-spoiQ

**Reply by FrankZTH** [reply]
Your url is not found
